README.md

```diff
@@ -0,0 +1,9 @@
+# Service Weaver Template
+
+This repository contains a basic template of a Service Weaver application that
+you can use via `gonew`.
+
```

Comment:
Can you also add some language as to why someone would want to use this repo? e.g.

```
This template will allow you to get started on using Service Weaver more easily. 
```

Reply:
Done.